formats/capabilities/SPEC_1.0.md

```diff
@@ -0,0 +1,91 @@
+# Capability format
+
+<dl>
+	<dt>Namespace</dt><dd>`capabilities`</dd>
+	<dt>Version</dt><dd>1.0</dd>
+</dl>
+
+This is an application format for continuous sharing and minting of capability information
+
+## Subspace
+
+Capabilities are by an entity, and are therefore published under the issuing entities subspace.
+
+## Public capabilities
+
+The knowledge that you've issued capabilities to the receiver will be public for all
+
+`/capabilities/1.0/public/{receiver-pubkey}/{id}`
+
+## Private Capabilities
+
+You may want to publish these entries to all (e.g. You'd like help from everyone in the network to distribute, because you don't have a direct peer with the recipient),
+while not allowing people to know who you're actually granting these to.
+
+More formally, given the issuer, a path a capability is published under, and a proposed receiver, it should not be possible for anyone else to know if that is the correct receiver.
+
+For such cases instead of publishing these under the pubkey of the receiving entity, they are instead published under `encrypt(receiver, concat(receiver-pubkey, nonce))`. The receiver must attempt to decrypt all paths until they find their one. The issuer should save and use the same nonce in future so this only needs to happen once.
+
+`/capabilities/1.0/private/{encrypt(receiver, concat(receiver-pubkey, nonce))}/{id}`
+
+## Capability document IDs
+
+You may want one, or many, depending on the application, if you want to coordinate between multiple client devices etc. These are opaque to the receiver, who should decode all documents under the path prefix.
+
+A suggested scheme for this would be using `/1`, `/2` etc. If you're merely extending the lifetime of an existing cap, then you can write to the same path. It doesn't matter if you have two clients that independently do this. If you want to change the contents, you should increment the number, and tombstone older entries. This behaviour doesn't handle the same user making changes to the same receiver's caps concurrently on two devices they own - only one will win.
```

Comment:
Not quite getting either what exactly you describe here, nor which problem it solves =(
Which might very well be attributed to me being exhausted, but at the very least, it points to an opportunity to clarify the writing a bit.

Reply:
Fair, will try and elaborate and reword. This relates to our discussion on Discord on multiple client apps using the same keypair to run this extension process, and what happens if they conflict. I think some more explicit examples would likely help. (Hmm, maybe a diagram? Will give it a go)